tsan_arangodb_suppressions.txt

```diff
@@ -27,3 +27,19 @@ race:AllowImplicitCollectionsSwitcher
 
 # Known false-positive, see https://github.com/google/sanitizers/issues/953
 race:absl/synchronization/internal/waiter.cc
+
+# TODO Fix known thread leaks
+thread:ClusterFeature::startHeartbeatThread
+thread:CacheManagerFeature::start
+
+# TODO Fix lock order inversion
+deadlock:consensus::Agent::setPersistedState
```

Comment:
Should be fixed cluster team.
Agent call State
but sometimes
State call Agent